spanner/api/Spanner/CreateInstance.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_create_instance]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Spanner.Admin.Instance.V1;
+using Google.Cloud.Spanner.Common.V1;
+using Google.LongRunning;
+using System;
+
+public class CreateInstance
+{
+    public static Instance SpannerCreateInstance(string projectId, string instanceId)
+    {
+        // Create the InstanceAdminClient instance.
+        InstanceAdminClient instanceAdminClient = InstanceAdminClient.Create();
+
+        // Initialize request parameters.
+        Instance instance = new Instance
+        {
+            InstanceName = InstanceName.FromProjectInstance(projectId, instanceId),
+            ConfigAsInstanceConfigName =
+                InstanceConfigName.FromProjectInstanceConfig(projectId, "regional-us-central1"),
+            DisplayName = "This is a display name.",
+            NodeCount = 2,
+            Labels =
+            {
+                { "cloud_spanner_samples", "true" },
+            }
+        };
+        ProjectName parentAsProjectName = ProjectName.FromProject(projectId);
+
+        // Make the CreateInstance request.
+        Operation<Instance, CreateInstanceMetadata> response =
+            instanceAdminClient.CreateInstance(parentAsProjectName, instanceId, instance);
+
+        Console.WriteLine("Waiting for the operation to finish.");
+
+        // Poll until the returned long-running operation is complete.
+        Operation<Instance, CreateInstanceMetadata> completedResponse =
+            response.PollUntilCompleted();
```

Comment:
My bad with ThrowOnAnyErro(). That's something we write for BigQuery, manually. For grpc long running operations the correct way of checking errors is how it's being done in this sample.